app/src/main/java/org/mozilla/fenix/home/sessioncontrol/viewholders/pocket/PocketStoriesComposables.kt

```diff
@@ -102,11 +107,15 @@ fun PocketStories(
             Column(Modifier.padding(end = if (rowIndex < gridLength - 1) 8.dp else 0.dp)) {
                 for (index in 0 until gridLength) {
                     columnItems.getOrNull(index)?.let { story ->
+                        val uri = Uri.parse(story.url)
```

Comment:
It could be better to have this as an argument to `onExternalLinkClicked` instead of a separate property, in this only calculating the link when the story is clicked. Everytime the story is clicked. So .. not really sure what would be best.